apps/codeforafrica/src/payload/blocks/Posts.js

```diff
@@ -73,6 +73,16 @@ const Posts = {
           type: "text",
           defaultValue: "Read More",
         },
+        {
```

Comment:
@kelvinkipruto  thinking out loud, is recent~Stories expected to be fixed by post? why not fetch from the collection ordering by publishedOn ?

Reply:
@koechkevin I'm already fetching [ordered by publishedOn](https://github.com/CodeForAfrica/ui/blob/95c9b955435e94ab67696b83c8e6ede4c80cc393/apps/codeforafrica/src/lib/data/pagify/post.js#L8-L15). This is just the title for the Recent Stories